install_caffe.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+sudo apt-get install protobuf-compiler libboost-all-dev libgflags-dev libgoogle-glog-dev libhdf5-serial-dev libleveldb-dev liblmdb-dev libsnappy-dev libopencv-dev libyaml-dev
+
+git clone https://github.com/BVLC/caffe.git ../caffe
+
+cp Makefile.config.example ../caffe/Makefile.config
+
+cd ../caffe
```

Comment:
Have a look at similar instructions in `install-deeplearning-libraries.sh`.

You can follow the pattern:

```bash
(cd caffee && run command)
```

to run the command inside the caffee folder and restore the CWD automatically to the previous state once done with that command.